protocol/describeclientquotas/describeclientquotas.go

```diff
@@ -0,0 +1,50 @@
+package describeclientquotas
+
+import "github.com/segmentio/kafka-go/protocol"
+
+func init() {
+	protocol.Register(&Request{}, &Response{})
+}
+
+type Request struct {
```

Comment:
I believe you'll need a field to indicate v1 is a flexible message like
https://github.com/segmentio/kafka-go/blob/5e7b724b8a23e2dc5adfb508c00dec5307719692/protocol/addoffsetstotxn/addoffsetstotxn.go#L12